content/html/concepts/elements/terms/label/label.md

```diff
@@ -0,0 +1,97 @@
+# Label
```

Comment:
The metadata for this entry is missing. You can look at the source for one of the other HTML entries to see what it looks like.  Also review the [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and the [entry template](https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md).